tests/covariance_matrix_05.output

```diff
@@ -0,0 +1 @@
+(6,0)
```

Comment:
I'll note that what you have here is really a *real* number, not an imaginary number.